Client/Frontend/Browser/BrowserViewController/BrowserViewController+URLBarDelegate.swift

```diff
@@ -121,23 +121,31 @@ extension BrowserViewController: URLBarDelegate {
                     self.legacyShowSettings(deeplink: .contentBlocker)
                 }
             }
-
-            let etpVC = EnhancedTrackingProtectionMenuVC(viewModel: etpViewModel)
-            if UIDevice.current.userInterfaceIdiom == .phone {
-                etpVC.modalPresentationStyle = .custom
-                etpVC.transitioningDelegate = self
+            if CoordinatorFlagManager.isEtpCoordinatorEnabled {
+                DispatchQueue.main.async {
+                    self.navigationHandler?.showEnhancedTrackingProtection()
+                }
             } else {
-                etpVC.asPopover = true
-                etpVC.modalPresentationStyle = .popover
-                etpVC.popoverPresentationController?.sourceView = urlBar.locationView.trackingProtectionButton
-                etpVC.popoverPresentationController?.permittedArrowDirections = .up
-                etpVC.popoverPresentationController?.delegate = self
+                self.legacyShowEnhancedTrackingProtection(viewModel: etpViewModel)
             }
-
-            TelemetryWrapper.recordEvent(category: .action, method: .press, object: .trackingProtectionMenu)
-            self.present(etpVC, animated: true, completion: nil)
         }
     }
+    private func legacyShowEnhancedTrackingProtection(viewModel: EnhancedTrackingProtectionMenuVM) {
+        let etpVC = EnhancedTrackingProtectionMenuVC(viewModel: viewModel)
+        if UIDevice.current.userInterfaceIdiom == .phone {
+            etpVC.modalPresentationStyle = .custom
+            etpVC.transitioningDelegate = self
+        } else {
+            etpVC.asPopover = true
+            etpVC.modalPresentationStyle = .popover
+            etpVC.popoverPresentationController?.sourceView = urlBar.locationView.trackingProtectionButton
+            etpVC.popoverPresentationController?.permittedArrowDirections = .up
+            etpVC.popoverPresentationController?.delegate = self
+        }
+
+        TelemetryWrapper.recordEvent(category: .action, method: .press, object: .trackingProtectionMenu)
```

Comment:
We could move the telemetry atop the `if CoordinatorFlagManager.isEtpCoordinatorEnabled` to ensure we still send the telemetry even when the coordinator is enabled.